apps/website/src/pages/docs/tabs.mdx

```diff
@@ -9,27 +9,92 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tabs' />
-
 <LiveExample src='Tabs.main.tsx'>
   <AllExamples.TabsMainExample client:load />
 </LiveExample>
 
-## Usage
+### Composition API
+
+You can build Tabs with subcomponents that are fully customizable.
+
+We provide:
+
+- `Tabs.Wrapper`
+- `Tabs.List`
+- `Tabs.Tab`
+- `Tabs.TabIcon`
+- `Tabs.TabLabel`
+- `Tabs.TabDescription`
+- `Tabs.Actions`
+- `Tabs.Panel`
+
+`Tabs.Tab` and `Tabs.Panel` can be associated with each other by passing them the same `value`.
```

Comment:
We should also explain the `value` on `onValueChange` props (uncontrolled vs controlled) somewhere on this page

Reply:
Added controlled state section